misc/scripts/models-as-data/bulk_generate_mad.py

```diff
@@ -36,8 +39,23 @@ class Project(TypedDict):
     """
 
     name: str
-    git_repo: str
+    git_repo: NotRequired[str]
     git_tag: NotRequired[str]
+    with_sinks: NotRequired[bool]
+    with_sinks: NotRequired[bool]
+    with_summaries: NotRequired[bool]
+
+
+def shouldGenerateSinks(project: Project) -> bool:
+    return project.get("with_sinks", False)
+
+
+def shouldGenerateSources(project: Project) -> bool:
+    return project.get("with_sources", False)
```

Comment:
What about having this default to `True`? I think it's nicer to have that default on the assumption that the model generator does good and useful things, and you'd only have to opt out in specific cases?

Reply:
Good idea. I've fixed this in 7cb9024cc620a2ecd9b7f41e8a3224e56c0cb8f3